### PR TITLE
Add custom metadata message for lens position (this time to correct branch)

### PR DESCRIPTION
**Pull request by jdavidberger**
Also threw in everything available on an ImgFrame.

Note that I could not test on ROS2.

### CODE REVIEW

**Comment by saching13** [comment]
```
  Error processing 'float32 LensPosition' of
  'depthai_ros_msgs/CameraMetadata': ''LensPosition' is an invalid field
  name.  It should have the pattern '^(?!.*__)(?!.*_$)[a-z][a-z0-9_]*$''

```

Can you change the field names to `snake_case` ? PascalCase doesn't satisfy ros2 convention.

File: depthai_bridge/include/depthai_bridge/BridgePublisher.hpp [hunk 1]
```diff
@@ -26,6 +26,7 @@
     #include <boost/make_shared.hpp>
     #include <boost/shared_ptr.hpp>
 
+    #include "depthai_ros_msgs/CameraMetadata.h"
```
Comment (saching13) [inline depthai_bridge/include/depthai_bridge/BridgePublisher.hpp:29]: Can you add this to `#ifdef IS_ROS2` part ?
`#include "depthai_ros_msgs/msg/CameraMetadata.hpp"`

**Review by saching13 — CHANGES_REQUESTED** [review]
I think after adding the ROS2 header we are good to merge.

**Comment by jdavidberger** [comment]
I have another larger PR that I'm going to push in a little bit; is there a quick ROS2 check docker build or something I could run to sanity check?

**Pull request merged**